docs/concepts/model_bundles.md

```diff
@@ -5,16 +5,45 @@ are created by packaging a model up into a deployable format.
 
 ## Creating Model Bundles
 
-There are two methods for creating model bundles:
-[`create_model_bundle`](/api/client/#launch.client.LaunchClient.create_model_bundle)
+There are three methods for creating model bundles:
+[`create_model_bundle_from_callable_v2`](/api/client/#launch.client.LaunchClient.create_model_bundle_from_callable_v2)
 and
-[`create_model_bundle_from_dirs`](/api/client/#launch.client.LaunchClient.create_model_bundle_from_dirs).
-The former directly pickles a user-specified `load_predict_fn`, a function which
+[`create_model_bundle_from_dirs_v2`](/api/client/#launch.client.LaunchClient.create_model_bundle_from_dirs_v2).
+[`create_model_bundle_from_runnable_image_v2`](/api/client/#launch.client.LaunchClient.create_model_bundle_from_runnable_image_v2).
+The first directly pickles a user-specified `load_predict_fn`, a function which
 loads the model and returns a `predict_fn`, a function which takes in a request.
-The latter takes in directories containing a `load_predict_fn` and the
+The second takes in directories containing a `load_predict_fn` and the
 module path to the `load_predict_fn`.
+The third takes a Docker image that contains an entrypoint that starts a process
```

Comment:
I believe this should say that the user needs to provide a command. IIRC that's explicit in the API, right?